packages/modules/src/modules/banners/momentTemplate/components/MomentTemplateBannerVisual.tsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import { css } from '@emotion/react';
+import { ImageSettings } from '../settings';
+import { from } from '@guardian/src-foundations/mq';
+
+// ---- Component ---- //
+
+interface MomentTemplateBannerVisualProps {
+    settings: ImageSettings;
+}
+
+export const MomentTemplateBannerVisual = ({
+    settings,
+}: MomentTemplateBannerVisualProps): JSX.Element => (
+    <div css={container}>
+        <picture>
```

Comment:
There's a `ResponsiveImage component` in `modules/shared/ResponsiveImage.tsx` - it may be worth investigating if it could be used here?

Reply:
Ooo this looks perfect! Will do 👍